src/cli/cmds/fleet.toit

```diff
@@ -549,23 +549,23 @@ create-fleet-commands config/Config cache/Cache ui/Ui -> List:
 
   recovery-add-cmd := cli.Command "add"
       --help="""
-          Add a recovery server to this fleet.
+          Add a recovery URL to this fleet.
           """
       --rest=[
         cli.Option "url"
             --help="The URL of the recovery server."
             --required,
       ]
       --examples=[
-        cli.Example "Add a recovery server 'https://recovery.toit.io':"
+        cli.Example "Add a recovery URL 'https://recovery.toit.io':"
```

Comment:
This seems like a poor example. For the fleet commands, the URL should probably be more specific (include a fleet id, etc.).

Reply:
done.